app/page.tsx

```diff
@@ -3,13 +3,16 @@ import Image from 'next/image'
 import { Button } from 'greenhouse-react-ui'
 import { useRouter } from 'next/navigation'
 import { useMediaQueries } from '@react-hook/media-query'
+import React from 'react'
+import Loading from './components/Loading'
 
 export default function Home() {
   const router = useRouter()
   const { matches } = useMediaQueries({
     md: 'only screen and (min-width: 768px)',
     xxl: 'only screen and (min-width: 1536px)',
   })
+  const [isLoading, setLoading] = React.useState(true)
```

Comment:
Please change the default loading state to `false`. 

It is not changing to false anywhere and hence showing the loader on home page forever.

```diff
- const [isLoading, setLoading] = React.useState(true)
+ const [isLoading, setLoading] = React.useState(false)
```

Reply:
Oops! Sorry! I set it to true for easier testing of the styles, and I forgot to change it back.

Reply:
yes, that's one of "THE" common mistake

happens with everyone 😅